draft-ietf-ppm-dap.md

```diff
@@ -2477,6 +2476,17 @@ disagrees). See
 [#22](https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/22) for
 discussion.
 
+## Verification key requirements {#verification-key}
+
+The verification key for a task SHOULD be chosen before any reports are generated.
+It SHOULD be fixed for the lifetime of the task and cannot be rotated. One way
+to ensure this is to include the verification key in a derivation of the task ID.
+
+To ensure that the security proofs for Prio3 hold, the verification key MUST be
+chosen independently of the generated reports. This can be achieved as
+recommended above. It would also be possible for the aggregators to use a coin
+flipping protocol if key rotation is necessary.
```

Comment:
I don't think "coin flipping protocol" means much to most readers. Do we have a concrete suggestion here? If not, I think we should either cut this suggestion or add an OPEN ISSUE for tracking discussion on this. My preference would be the former.

Reply:
> if key rotation is necessary.

We say above that verification keys cannot be rotated, so it seems like this sentence can be deleted. And I agree with Chris: the specification is to be read by implementers who aren't necessarily cryptography experts and need concrete advice on how to do this. If I remember right, it's sufficient for the Leader to randomly generate a verification key with a CSPRNG and then transmit it to the Helper over a secure channel (like TLS). If we want to provide a concrete recommendation, does that suffice?

Reply:
That would be sufficient, and I think it would be worth RECOMMENDING this. However this is kind of separate from what @simon-friedberger is suggesting here: If we specified some sort of protocol for exchanging the key whereby each Aggregator committed to and provided some entropy, then we might be able weaken the "don't rotate while running a task" requirement.

Reply:
Yeah, I was under the impression that VDAF-04 had already forbidden verification key rotation, but that's not the case. Then I agree, we should leave the door open for coin-flipping schemes.

Reply:
Would a reference be sufficient clarification? Maybe this?
> @inproceedings{Blum1981CoinFB,
  title={Coin Flipping by Telephone.},
  author={Manuel Blum},
  booktitle={Annual International Cryptology Conference},
  year={1981}
}

Or we could add a description of such a protocol, @cjpatton proposed one in #161.

Reply:
The Blum paper is the first crypto paper I ever read! Very cool, but very outdated. Moreover, I am not ready to endorse the solution we sketched in #161. I think what we would need here is a concrete proposal + security analysis.

I really think we should do nothing here until we've had a chance to actually work it out.

Reply:
Removed.